tuna/miopen/db/session.py

```diff
@@ -71,6 +72,6 @@ def add_new_session(self, args, worker):
     if hasattr(args, 'miopen_v') and args.miopen_v:
       self.miopen_v = args.miopen_v
     else:
-      self.miopen_v = worker.get_miopen_v()
+      self.miopen_v = get_miopen_v(self)
```

Comment:
The worker that is submitted to this function needs to be a derivative of FinClass. Specify that in the type hint, and leave the call to worker.get_miopen_v.